Cooper/ClientApp/src.v2/app/components/user-connections-list/user-connections-list.component.css

```diff
@@ -1,7 +1,7 @@
   .user-interaction-card {   
     object-fit: cover;
     width: 100%;
-    height: 292px;
+    height: 280px;
```

Comment:
Refactoring that file is not related to issue. It overlaps the changes that were made in #173 (Nickname bug on profile) pull-request